packages/frontend/src/components/modules/documents/Faq/Faq.module.scss

```
@@ -0,0 +1,66 @@
+.title {
+  @include headline5;
+  position: relative;
+  margin: 59px 0 0 100px;
+
+  @include bp-1059 {
+    width: 688px;
+    padding-bottom: 24px;
+    margin: 0 auto;
+    margin-top: 92px;
+  }
+  @include bp-767 {
+    max-width: 100%;
+  }
+
+  @include bp-359 {
+    padding-left: 16px;
+  }
+
+  &::after {
+    position: absolute;
+    top: -80%;
+    left: 90%;
+    width: 58px;
+    height: 61px;
+    content: '';
+    background-image: url('/images/Pic-question.svg');
+    @include bp-1059 {
+      top: -37%;
+      left: 90%;
+    }
+    @include bp-359 {
+      left: 81%;
+    }
+  }
+}
+
+.wrapper-box {
+  display: flex;
+  flex-direction: column;
+  // width: 844px;
+  max-height: 730px;
+  margin: 24px 0 0 100px;
+  transition: all 500ms ease-in;
+  @include bp-1279 {
+    max-width: 688px;
+    margin: 0 auto;
+    margin: 24px 0 0 100px;
+  }
+  @include bp-1059 {
+    margin: 0 auto;
+    margin-bottom: 18%;
+  }
+
+  @include bp-359 {
+    margin-bottom: 120%;
```

А здесь тогда можно тоже полной записью маржина изменить. Если ты из-за этой записи выше писал margin и margin-bottom
```suggestion
    margin: 0 auto 120%;
```

---

А зачем ты вообще margin-bottom задаешь в %?
Отчего проценты берешь и как на макете их вычислил, от чего?
Я думаю для нижнего маржина лучше пиксели использовать...